config.go

```diff
@@ -65,13 +65,19 @@ type DMuxConfigSetting struct {
 	FilePath string
 }
 
+type Limits struct{
+	MaxTopics  int			`json:"topics"`
+	MaxPart    int			`json:"partitions_per_topic"`
+}
+
 //DmuxConf hold config data
 type DmuxConf struct {
-	Name      string     `json:"name"`
-	DMuxItems []DmuxItem `json:"dmuxItems"`
+	Name      string     	`json:"name"`
+	DMuxItems []DmuxItem 	`json:"dmuxItems"`
 	// DMuxMap    map[string]KafkaHTTPConnConfig `json:"dmuxMap"`
-	MetricPort int	     `json:"metric_port"`
+	MetricPort int	     	`json:"metric_port"`
 	Logging logging.LogConf `json:"logging"`
+	MaxLimits Limits		`json:"limits"`
```

Comment:
let's group metric related configs under metric key & rename `MaxLimits` to `OffsetTrackerLimits` accordingly update json value too